client/app/components/game-client.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
+    GameChannel.on('event', payload => {
+      this.putMessage({
+        server: {
```

Comment:
addClient/addServer is good idea as well almost did that my self, but was trying to finish it. Can we come back to this in another round perhaps? 

See my note on the template though, I do think the css class will be needed though, but there is a reason I used client: and server: because I needed more than just css at the time.

That said I think the server will need to include colors etc so cssClass will be needed I think if nothing else to change colors and other things.